transcendence-app/src/main.ts

```diff
@@ -19,6 +19,7 @@ async function bootstrap() {
       transformOptions: {
         enableImplicitConversion: true,
       },
+      forbidUnknownValues: true,
```

Comment:
The [docs](https://github.com/typestack/class-validator#passing-options) recommend this option.